packages/cozy-harvest-lib/src/helpers/queries.js

```diff
@@ -84,3 +84,11 @@ export const buildCountTriggersQuery = () => ({
     fetchPolicy: defaultFetchPolicy
   }
 })
+
+export const buildAppsRegistryMaintenance = () => ({
+  definition: Q('io.cozy.apps_registry').getById('maintenance'),
+  options: {
+    as: 'io.cozy.apps_registry/maintenance',
+    fetchPolicy: defaultFetchPolicy
+  }
+})
```

Comment:
maybe you can rework this hook in order to use it here? (and make it use your new mechanism) 

https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/hooks/useAppsInMaintenance.jsx

Reply:
This is a nit for this PR since you can't use the hook directly. 

But I think you should update the hook anyway in order to optimise it by using a useQuery call ;)

Reply:
Very good idea to optimise performance at home. I changed the hook to use the same request ([PR](https://github.com/cozy/cozy-client/pull/1407))